### PR TITLE
docs: update issue template

### PR DESCRIPTION
**Pull request by dsanders11**
#### Description of Change
The issue template shown in the docs was outdated.

#### Checklist
- [x] PR description included and stakeholders cc'd

#### Release Notes

Notes: none

### CODE REVIEW

File: docs/development/issues.md [hunk 1]
```diff
@@ -36,41 +36,58 @@ When opening a new issue in the [`electron/electron` issue tracker](https://gith
...
 
 ```markdown
-<!--
-Thanks for opening an issue! A few things to keep in mind:
-
-- The issue tracker is only for bugs and feature requests.
-- Before reporting a bug, please try reproducing your issue against
-  the latest version of Electron.
-- If you need general advice, join our Slack: http://atom-slack.herokuapp.com
+<!--  As an open source project with a dedicated but small maintainer team, it can sometimes take a long time for issues to be addressed so please be patient and we will get back to you as soon as we can.
```
Comment (ckerr) [inline docs/development/issues.md:39]:
```suggestion
<!--  Please be patient! We are a small team.
```

File: docs/development/issues.md [hunk 2]
```diff
@@ -36,41 +36,58 @@ When opening a new issue in the [`electron/electron` issue tracker](https://gith
...
 -->
 
-* Electron version:
-* Operating system:
+### Preflight Checklist
+<!-- Please ensure you've completed the following steps by replacing [ ] with [x]-->
```
Comment (ckerr) [inline docs/development/issues.md:43]:
```suggestion
<!-- Please replace [ ] with [x] after completing each step -->
```

File: docs/development/issues.md [hunk 3]
```diff
@@ -36,41 +36,58 @@ When opening a new issue in the [`electron/electron` issue tracker](https://gith
...
+<!-- A clear and concise description of what actually happened. -->
 
+### To Reproduce
 <!--
+Your best chance of getting this bug looked at quickly is to provide an example.
+-->
 
-Your best chance of getting this bug looked at quickly is to provide a REPOSITORY that can be cloned and run.
+<!--
+For bugs that can be encapsulated in a small experiment, you can use Electron Fiddle (https://github.com/electron/fiddle) to publish your example to a GitHub Gist and link it your bug report.
```
Comment (ckerr) [inline docs/development/issues.md:70]:
```suggestion
For bugs that can be encapsulated in a small experiment, please use Electron Fiddle (https://github.com/electron/fiddle) to publish your example to a GitHub Gist and link it your bug report.
```

File: docs/development/issues.md [hunk 4]
```diff
@@ -36,41 +36,58 @@ When opening a new issue in the [`electron/electron` issue tracker](https://gith
...
+<!--  As an open source project with a dedicated but small maintainer team, it can sometimes take a long time for issues to be addressed so please be patient and we will get back to you as soon as we can.
 -->
 
-* Electron version:
-* Operating system:
+### Preflight Checklist
+<!-- Please ensure you've completed the following steps by replacing [ ] with [x]-->
 
-### Expected behavior
+* [ ] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
```
Comment (ckerr) [inline docs/development/issues.md:45]:
```suggestion
* [ ] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md).
```

File: docs/development/issues.md [hunk 5]
```diff
@@ -36,41 +36,58 @@ When opening a new issue in the [`electron/electron` issue tracker](https://gith
...
+* [ ] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
```
Comment (ckerr) [inline docs/development/issues.md:46]:
```suggestion
* [ ] I will follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md).
```

File: docs/development/issues.md [hunk 6]
```diff
@@ -36,41 +36,58 @@ When opening a new issue in the [`electron/electron` issue tracker](https://gith
...
+* [ ] I have searched the issue tracker for an issue that matches the one I want to file, without success.
```
Comment (ckerr) [inline docs/development/issues.md:47]:
```suggestion
* [ ] I looked but did not find this issue in the [issue tracker](https://github.com/electron/electron/issues).
```

File: docs/development/issues.md [hunk 7]
```diff
@@ -36,41 +36,58 @@ When opening a new issue in the [`electron/electron` issue tracker](https://gith
...
 
-<!-- What do you think should happen? -->
+### Issue Details
 
-### Actual behavior
+* **Electron Version:**
+  * <!-- (output of `node_modules/.bin/electron --version`) e.g. 4.0.3 -->
+* **Operating System:**
+  * <!-- (Platform and Version) e.g. macOS 10.13.6 / Windows 10 (1803) / Ubuntu 18.04 x64 -->
```
Comment (ckerr) [inline docs/development/issues.md:54]:
```suggestion
  * <!-- (Platform and Version) e.g. macOS 10.13.6 / Windows 10 (1803) / Ubuntu 20.04 x64 -->
```

File: docs/development/issues.md [hunk 8]
```diff
@@ -36,41 +36,58 @@ When opening a new issue in the [`electron/electron` issue tracker](https://gith
...
+* **Last Known Working Electron version:**
+  * <!-- (if applicable) e.g. 3.1.0 -->
```
Comment (ckerr) [inline docs/development/issues.md:56]:
```suggestion
  * <!-- (if applicable) e.g. 10.1.5 -->
```

File: docs/development/issues.md [hunk 9]
```diff
@@ -36,41 +36,58 @@ When opening a new issue in the [`electron/electron` issue tracker](https://gith
...
 
-<!-- What actually happens? -->
+### Expected Behavior
+<!-- A clear and concise description of what you expected to happen. -->
```
Comment (ckerr) [inline docs/development/issues.md:59]:
```suggestion
<!-- Describe what you expected to happen. -->
```

File: docs/development/issues.md [hunk 10]
```diff
@@ -36,41 +36,58 @@ When opening a new issue in the [`electron/electron` issue tracker](https://gith
...
 
-### How to reproduce
+### Actual Behavior
+<!-- A clear and concise description of what actually happened. -->
```
Comment (ckerr) [inline docs/development/issues.md:62]:
```suggestion
<!-- Describe what actually happened. -->
```

File: docs/development/issues.md [hunk 11]
```diff
@@ -36,41 +36,58 @@ When opening a new issue in the [`electron/electron` issue tracker](https://gith
...
 
+### To Reproduce
 <!--
+Your best chance of getting this bug looked at quickly is to provide an example.
+-->
```
Comment (ckerr) [inline docs/development/issues.md:67]: These paragraphs could probably all go in one `<!--` ... `-->` block to reduce the visual nosiness

File: docs/development/issues.md [hunk 12]
```diff
@@ -36,41 +36,58 @@ When opening a new issue in the [`electron/electron` issue tracker](https://gith
...
-  $ npm install
-  $ npm start || electron .
+$ git clone $YOUR_URL -b $BRANCH
+$ npm install
+$ npm start || electron .
 
 -->
+
+### Screenshots
+<!-- If applicable, add screenshots to help explain your problem. -->
```
Comment (ckerr) [inline docs/development/issues.md:87]:
```suggestion
<!-- If applicable, add screenshots to help explain the issue. -->
```

File: docs/development/issues.md [hunk 13]
```diff
@@ -36,41 +36,58 @@ When opening a new issue in the [`electron/electron` issue tracker](https://gith
...
+
+### Additional Information
+<!-- Add any other context about the problem here. -->
```
Comment (ckerr) [inline docs/development/issues.md:90]:
```suggestion
<!-- Add any other context about the issue here. -->
```

File: docs/development/issues.md [hunk 14]
```diff
@@ -36,41 +36,58 @@ When opening a new issue in the [`electron/electron` issue tracker](https://gith
...
+For bugs that can be encapsulated in a small experiment, you can use Electron Fiddle (https://github.com/electron/fiddle) to publish your example to a GitHub Gist and link it your bug report.
+-->
 
-You can fork https://github.com/electron/electron-quick-start and include a link to the branch with your changes.
+<!--
+If Fiddle is insufficient to produce an example, please provide an example REPOSITORY that can be cloned and run. You can fork electron-quick-start (https://github.com/electron/electron-quick-start) and include a link to the branch with your changes.
+-->
 
+<!--
 If you provide a URL, please list the commands required to clone/setup/run your repo e.g.
```
Comment (ckerr) [inline docs/development/issues.md:78]:
```suggestion
If you provide a URL, please list the steps to use your repo e.g.
```

**Review by ckerr — COMMENTED** [review]
I really like the way this adds more structure to the template.

1. IMO it would make sense to reconcile this file with the template in `~/.github/ISSUE_TEMPLATE/`. (Actually... since everyone automatically sees the latter when they file a ticket, maybe we should focus on that one and remove this one from docs? I think the reason this one exists is because it predates the other.)

2. I suspect that people ignore instructions that are too long, so I like the issue template to be short and clear. To that end, most of my suggested changes are word golf -- and are therefore subjective, so feel free to push back if you like.

**Comment by dsanders11** [comment]
> 1. IMO it would make sense to reconcile this file with the template in `~/.github/ISSUE_TEMPLATE/`. (Actually... since everyone automatically sees the latter when they file a ticket, maybe we should focus on that one and remove this one from docs? I think the reason this one exists is because it predates the other.)

I think there may be some crossed wires on the communication here. I simply copy-and-pasted the one from `.github/ISSUE_TEMPLATE/Bug_Report.md` here. I assumed that originally it was an exact copy and they've since diverged.

As you said, it might make sense to remove from the docs, since it will inevitably always diverge.

**Review by jkleinsc — CHANGES_REQUESTED** [review]
I read through the comments on this PR and I think we should remove this section and replace it with a link to https://github.com/electron/electron/blob/master/.github/ISSUE_TEMPLATE/Bug_report.md

**Comment by dsanders11** [comment]
@jkleinsc, dropped the embedded text and put in a link instead.

**Review by jkleinsc — APPROVED** [review]

**Comment by jkleinsc** [comment]
Merging as CI failure unrelated to doc change.

**Pull request merged**